### DISCUSSION TITLE
[3.2] Usability FR: Improve visibility of drafts

### DISCUSSION THREAD

**Discussion by ghost** [craftcms/cms#8701] [Ideas]
When walking through the new drafts workflow with a couple of agencies i support, the main issue that popped up was that you can easily loose track of existing drafts.

So two simple suggestions: (maybe we overlooked something, it is already planned or considered plugin territory..)

* On the edit screen, existing drafts should always be visible, especially in a prominent way when the 'main' version is opened from the element index, in order to avoid multiple conflicting drafts. (They exist in the drop down, but that will be overlooked)
* The element indexes should be able to show existing drafts (like trashed items), including 'unsaved' drafts.

**Comment by mildlygeeky** [comment]
100% agree with this — loving 3.2 (wasn't able to really get into the beta / RC), but I can definitely see people being taken by surprise that they have created drafts.

Given that drafts have now become much more powerful in Craft, it feels like there should be some sort of visual indicator in the Draft dropdown that lets someone see at a glance how many drafts exist for the content they are viewing. Not sure it needs to be that drafts are always visible in their own dropdown, but maybe a number count of drafts (thinking something in a circle with a color background that will let it grab someone's attention) alongside the "Current" moniker.

**Comment by richhayler** [comment]
Having an option to add a “drafts” column to display the number of drafts for each entry on element index pages could also be a good way to keep tabs on entry drafts across all content.
